### PR TITLE
Add Trix and Turbo detection

### PR DESCRIPTION
**Pull request by xfoucron**
Websites :
- https://trix-editor.org/
- https://turbo.hotwired.dev/

Examples :
- https://trix-editor.org/
- https://turbo.hotwired.dev/

### CODE REVIEW

File: src/technologies/t.json [hunk 1]
```diff
@@ -42,6 +42,18 @@
...
     "website": "https://www.trisoshop.pl"
   },
+  "Trix": {
+    "cats": [
+      24
+    ],
+    "html": [
+      "<[a-z]+ [^>]*class=\"trix-container\"",
+      "<trix-editor"
+    ],
```
Comment (nurbek91) [inline src/technologies/t.json:52]: Let's remove the html and write it like this instead:
```
"js": {
      "Trix.VERSION": "^(.+)$\\;version:\\1"
    }
```
example: https://meucondo.co.ao/

File: src/technologies/t.json [hunk 2]
```diff
@@ -42,6 +42,18 @@
...
+    "website": "https://trix-editor.org",
+    "description": "Trix is an open-source project from Basecamp, the creators of Ruby on Rails. Millions of people trust their text to Basecamp, and we built Trix to give them the best possible editing experience.",
```
Comment (nurbek91) [inline src/technologies/t.json:54]: Trix is an open-source rich text editor from Basecamp.

File: src/technologies/t.json [hunk 3]
```diff
@@ -1930,6 +1942,18 @@
     "url": "^https?://(?:www\\.)?[^/]+\\.tumblr\\.com/",
     "website": "http://www.tumblr.com"
   },
+  "Turbo": {
+    "cats": [
+      92
+    ],
+    "website": "https://turbo.hotwired.dev/",
+    "description": "Turbo is a JavaScript framework for building fast web applications.",
+    "icon": "turbo.png",
```
Comment (nurbek91) [inline src/technologies/t.json:1951]: [Turbo.svg.zip](https://github.com/AliasIO/wappalyzer/files/7770424/Turbo.svg.zip)
svg instead png

**Review by nurbek91 — COMMENTED** [review]
Hey there, thanks for PR! Little changes required.

**Comment by AliasIO** [comment]
I've applied the changes, thanks!

**Pull request merged**